README.md

```diff
@@ -67,14 +66,15 @@ Got any questions? Want to talk to the maintainers?
 Our [Discord community](https://discord.gg/2QDjZkYunf) is a great place to get in touch with us, and
 we'd love to have you there.
 
-<a href="https://discord.gg/2QDjZkYunf" alt="Melt UI Discord community">
-<picture>
-  <source media="(prefers-color-scheme: dark)" srcset="https://invidget.switchblade.xyz/2QDjZkYunf">
-  <img alt="Melt UI Discord community" src="https://invidget.switchblade.xyz/2QDjZkYunf?theme=light">
-</picture>
-</a>
+[![Melt UI Discord community](https://invidget.switchblade.xyz/2QDjZkYunf?theme=dark)](https://discord.gg/2QDjZkYunf)
```

Comment:
This is not the same as the HTML version, it ignores light mode which is not desireable. We can keep the HTML version

Reply:
Resolved in the commit ef4a3506fda3ae1e03b3490b5a4cced6c0f6fd6c